python/py-graph-tool/Portfile

```diff
@@ -115,6 +133,7 @@ if {${name} ne ${subport}} {
     destroot.cmd    make
     destroot.destdir \
                     DESTDIR=${destroot}
+    destroot.target install
```

Comment:
Isn't `install` the default value for `destroot.target`?

Reply:
It's overwritten by PG python, so this is necessary here.

Reply:
> It's overwritten by PG python, so this is necessary here.

Well, would it be more accurate to say that this is necessary when `python.pep517` is enabled? Which is the default for 3.11 and later.

I'd need to dig into the upstream project a bit, before providing any official recommendations. But if you're working around that by setting `destroot.target`, etc, then perhaps it might be better to simply set `python.pep517` to `no`. (Which might allow you to eliminate at least a few overrides you're setting.)

However, I'll let our more Python-savvy folks chime in with their thoughts and recommendations.

CC: @jmroot
CC: @reneeotten

Reply:
> I'd need to dig into the upstream project a bit, before providing any official recommendations. But if you're working around that by setting `destroot.target`, etc, then perhaps it might be better to simply set `python.pep517` to `no`. (Which might allow you to eliminate at least a few overrides you're setting.)

That being said, I'm not overly concerned, so no need to hold up merging this.